shared/controller-api/src/main/java/io/pravega/shared/controller/tracing/RPCTracingTags.java

```diff
@@ -0,0 +1,63 @@
+/**
```

Comment:
Could these be added to the existing RequestTracker class instead of creating a new class ?

Reply:
i think its better organized if its kept in a dedicated class for these constants..